frontend/public/components/marketplace/marketplace-items.jsx

```diff
@@ -91,11 +96,14 @@ const setURLParams = params => {
   history.replace(`${url.pathname}${searchParams}`);
 };
 
-class MarketplaceTileViewPage extends React.Component {
+export class MarketplaceTileViewPage extends React.Component {
```

Comment:
Wrap this with the [`requireOperatorGroup()` HOC](https://github.com/openshift/console/blob/master/frontend/public/components/operator-lifecycle-manager/operator-group.tsx#L23).